.teamcity/settings/extensions.kt

```diff
@@ -17,7 +17,10 @@ fun BuildType.agentRequirement(os: Os, arch: Arch = Arch.AMD64) {
 fun toolchainConfiguration(os: Os) = listOf(
     "-Porg.gradle.java.installations.auto-detect=false",
     "-Porg.gradle.java.installations.auto-download=false",
-    """"-Porg.gradle.java.installations.paths=%${os.name}.java8.oracle.64bit%,%${os.name}.java11.openjdk.64bit%""""
+    """"-Porg.gradle.java.installations.paths=
+        |%${os.name}.java8.oracle.64bit%,
+        |%${os.name}.java11.openjdk.64bit%,
```

Comment:
💭 I left Java11 here, since without https://github.com/gradle/gradle-profiler/pull/579 merged we still use Java11 toolchains